### ISSUE TITLE
Release v1.55.5

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#6937]
:robot: A dedicated issue to run tests and deploy release [v1.55.5](https://github.com/deckhouse/deckhouse/milestone/268).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.55.5 | test-v1.55.5*` or a branch `main | release-1.55`
- `/e2e/run/<provider> git_ref_1 git_ref_2` will run e2e using provider and git_ref_1 and then switch to git_ref_2.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref_1` is a release-* or main branch
  - `git_ref_2` is a release-* or main branch
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `docker | containerd`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.24 | 1.25 | 1.26 | 1.27 | 1.28`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.55.5 | test-v1.55.5*` or a branch `main | release-1.55`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.25'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.20
/e2e/use/k8s/1.23

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.20 and 1.23 using image built from main branch.
```

```
/e2e/run/aws release-1.35 release-1.36
/e2e/use/cri/containerd
/e2e/use/k8s/1.23

This comment will create cluster in AWS using Deckhouse built from release-1.35 branch
and then switch to images built from release-1.36 branch.
'use' options are supported, cluster will use containerd and Kubernetes version 1.23.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.55
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [d2f284](https://github.com/deckhouse/deckhouse/commit/d2f284c88c49c8599093766183f03de55d794852) in branch **main**:
```
[docs] Refactor markdown rendering in descriptions (#6936)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ebfa16](https://github.com/deckhouse/deckhouse/commit/ebfa166045de591fe89fdc9539c8f13e91d6de43) in branch **main**:
```
[node-manager](capi) remove validating webhook for Node deletion (#6938)

Signed-off-by: Artem Kulyabin <artem.kulyabin@flant.com>
Co-authored-by: Artem Kulyabin <artem.kulyabin@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [874f62](https://github.com/deckhouse/deckhouse/commit/874f6250e49db2d90b5210ccb2e734ebaa5f1716) in branch **main**:
```
[global] fix enabledModules values (#6935)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c26a25](https://github.com/deckhouse/deckhouse/commit/c26a2569b58000b3f247ec50be0a19ad8a972a7e) in branch **main**:
```
[modules docs] Fix rendering of resources to avoid overlapping with the TOC (#6946)

Signed-off-by: Artem Kosulin <artem.kosulin@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [320367](https://github.com/deckhouse/deckhouse/commit/3203677d8031bb604687dae691fd292188967a01) in branch **main**:
```
Re-generate changelog v1.55.5 (#6939)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: nikiGGG <nikiGGG@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.55.5** is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [46e37f](https://github.com/deckhouse/deckhouse/commit/46e37f0841ab47337c7411c95ce50619c7940485) in branch **main**:
```
[global] fix modules order on operator-startup (#6952)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [fd9b75](https://github.com/deckhouse/deckhouse/commit/fd9b75b818c9f7922f339b557aaf80961599c957) in branch **main**:
```
Re-generate changelog v1.56.0 (#6942)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: z9r5 <z9r5@users.noreply.github.com>
```
